pytorch_lightning/trainer/logging.py

```diff
@@ -112,10 +112,6 @@ def process_output(self, output, train=False):
             num_gpus = self.num_gpus
             callback_metrics = self.reduce_distributed_output(callback_metrics, num_gpus)
 
-        for k, v in callback_metrics.items():
```

Comment:
why remove this?

Reply:
it’s done twice. this is the first time but it’s done again at the end of the method